accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -217,7 +220,16 @@ assignable to `Object`.
 
 It is not an error for the body of a `late` field to reference `this`.
 
-It is an error for a formal parameter to be declared `late`.
+It is an error for a variable to be declared as `late` in any of the following
+positions: in a formal parameter list of any kind; in a catch clause; in the
+variable binding section of a c-style `for` loop, a `for in` loop, an `await
+for` loop, or a `for element` in a collection literal.
+
+It is an error for the initializer expression of a `late` local variable to use
+a prefix `await` expression.
+
+It is an error for a class with a `const` constructor to have a `late final`
+field.
```

Comment:
Does this include `static` fields? Or the set of modifiers should be exactly `late final`?

Reply:
No. I'm guessing that "field" means "instance variable" here. A "field" is not a formally specified term.